FrameMetadata/Lets-Build-OTPublisher/Counter.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Counter.h
+//  Custom-Video-Driver
+//
+//  Created by Lucas Huang on 4/12/18.
+//  Copyright © 2018 TokBox, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
IMHO having two classes for the metadata is a little bit confusing. If you plan to embed a "Counter" and the location, You could have a `FrameMetadataGenerator` class where both things are generated

Reply:
We would remove location and ith count per our morning meeting.